example/profiler/profiler_imageiter.py

```diff
@@ -0,0 +1,27 @@
+import os
+os.environ["MXNET_CPU_WORKER_NTHREADS"] = "4"
```

Comment:
better you don't hard coded here. so I can run a batch to get result for different threads #.